init documentation for `HTMLSourceElement.{height,width}` (#32106)
mdn/content: files/en-us/web/api/htmlsourceelement/height/index.md

---
title: "HTMLSourceElement: height property"
short-title: height
slug: Web/API/HTMLSourceElement/height
page-type: web-api-instance-property
browser-compat: api.HTMLSourceElement.height
---

{{APIRef("HTML DOM")}}

> The **`height`** property of the {{domxref("HTMLSourceElement")}} interface reflects a non-negative number indicating the height of the image resource in CSS pixels.

Reviewer: ```suggestion
The **`height`** property of the {{domxref("HTMLSourceElement")}} interface is a non-negative number indicating the height of the image resource in CSS pixels.
```

> The property has a meaning only if the parent of current {{HTMLElement("source")}} element is a {{HTMLElement("picture")}} element.

Reviewer 2: ```suggestion
The property has a meaning only if the parent of the current {{HTMLElement("source")}} element is a {{HTMLElement("picture")}} element.
```

> The property has a meaning only if the parent of the current {{HTMLElement("source")}} element is a {{HTMLElement("picture")}} element.

Reviewer: ```suggestion
The property has an effect only if the parent of the current {{HTMLElement("source")}} element is a {{HTMLElement("picture")}} element.
```
